plugins/typo_squatter/plugin.spec.yaml

```diff
@@ -29,17 +30,20 @@ triggers:
         description: Query to match
         type: string
         required: false
+        example: example
```

Comment:
Can I have some context on this example? Should the value for example here be `example`

Reply:
If the query is given, only domains with a match will be returned. So for this example, all domains that contain the word `example` will be returned. I've added a better example for this input because we're using regular expressions here.